packages/broker/src/plugins/operator/MaintainTopologyHelper.ts

```diff
@@ -93,22 +93,27 @@ export class OperatorClient extends EventEmitter<OperatorClientEvents> {
             this.sponsorshipCountOfStream.set(streamId, sponsorshipCount)
             if (sponsorshipCount === 0) {
                 this.sponsorshipCountOfStream.delete(streamId)
-                this.emit("removeStakedStream", streamId, await this.contract.provider.getBlockNumber())
+                this.emit("removeStakedStream", streamId)
             }
+            latestBlock = await this.contract.provider.getBlockNumber()
         })
-        await this.pullStakedStreams()
+        
+        const initalStreams = await this.pullStakedStreams(latestBlock)
```

Comment:
I believe `latestBlock` would always equal `0` at this point due to any potential event handlers getting fired earliest after the synchronous portion of the `start` method has finished executing. So a potential bug here?

```suggestion
        const initalStreams = await this.pullStakedStreams(latestBlock)
```